packages/data-table/src/DataTable.tsx

```diff
@@ -0,0 +1,393 @@
+import React from 'react';
+import styled from 'styled-components';
+import {
+  useTable,
+  usePagination,
+  useSortBy,
+  useExpanded,
+  useRowSelect,
+} from 'react-table';
+import { Chevron } from './Chevron';
```

Comment:
I think this should have a Icon or Svg suffix, like `ChevronSvg` to be more specific of what kinds of components it is.